app/app.go

```diff
@@ -919,6 +920,73 @@ func (app *BandApp) setupUpgradeHandlers() {
 			return app.mm.RunMigrations(ctx, app.configurator, fromVM)
 		},
 	)
+
+	app.UpgradeKeeper.SetUpgradeHandler(
+		"v2_5",
+		func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+			hostParams := icahosttypes.Params{
+				HostEnabled: true,
+				AllowMessages: []string{
+					sdk.MsgTypeURL(&authz.MsgExec{}),
+					sdk.MsgTypeURL(&authz.MsgGrant{}),
+					sdk.MsgTypeURL(&authz.MsgRevoke{}),
+					sdk.MsgTypeURL(&banktypes.MsgSend{}),
+					sdk.MsgTypeURL(&banktypes.MsgMultiSend{}),
+					sdk.MsgTypeURL(&distrtypes.MsgSetWithdrawAddress{}),
+					sdk.MsgTypeURL(&distrtypes.MsgWithdrawValidatorCommission{}),
+					sdk.MsgTypeURL(&distrtypes.MsgFundCommunityPool{}),
+					sdk.MsgTypeURL(&distrtypes.MsgWithdrawDelegatorReward{}),
+					sdk.MsgTypeURL(&feegrant.MsgGrantAllowance{}),
+					sdk.MsgTypeURL(&feegrant.MsgRevokeAllowance{}),
+					sdk.MsgTypeURL(&govv1beta1.MsgVoteWeighted{}),
+					sdk.MsgTypeURL(&govv1beta1.MsgSubmitProposal{}),
+					sdk.MsgTypeURL(&govv1beta1.MsgDeposit{}),
+					sdk.MsgTypeURL(&govv1beta1.MsgVote{}),
+					sdk.MsgTypeURL(&group.MsgCreateGroupPolicy{}),
+					sdk.MsgTypeURL(&group.MsgCreateGroupWithPolicy{}),
+					sdk.MsgTypeURL(&group.MsgCreateGroup{}),
+					sdk.MsgTypeURL(&group.MsgExec{}),
+					sdk.MsgTypeURL(&group.MsgLeaveGroup{}),
+					sdk.MsgTypeURL(&group.MsgSubmitProposal{}),
+					sdk.MsgTypeURL(&group.MsgUpdateGroupAdmin{}),
+					sdk.MsgTypeURL(&group.MsgUpdateGroupMembers{}),
+					sdk.MsgTypeURL(&group.MsgUpdateGroupMetadata{}),
+					sdk.MsgTypeURL(&group.MsgUpdateGroupPolicyAdmin{}),
+					sdk.MsgTypeURL(&group.MsgUpdateGroupPolicyDecisionPolicy{}),
+					sdk.MsgTypeURL(&group.MsgUpdateGroupPolicyMetadata{}),
+					sdk.MsgTypeURL(&group.MsgVote{}),
+					sdk.MsgTypeURL(&group.MsgWithdrawProposal{}),
+					sdk.MsgTypeURL(&oracletypes.MsgActivate{}),
+					sdk.MsgTypeURL(&oracletypes.MsgCreateDataSource{}),
+					sdk.MsgTypeURL(&oracletypes.MsgCreateOracleScript{}),
+					sdk.MsgTypeURL(&oracletypes.MsgEditDataSource{}),
+					sdk.MsgTypeURL(&oracletypes.MsgEditOracleScript{}),
+					sdk.MsgTypeURL(&oracletypes.MsgReportData{}),
+					sdk.MsgTypeURL(&oracletypes.MsgRequestData{}),
+					sdk.MsgTypeURL(&stakingtypes.MsgEditValidator{}),
+					sdk.MsgTypeURL(&stakingtypes.MsgDelegate{}),
+					sdk.MsgTypeURL(&stakingtypes.MsgUndelegate{}),
+					sdk.MsgTypeURL(&stakingtypes.MsgBeginRedelegate{}),
+					sdk.MsgTypeURL(&stakingtypes.MsgCreateValidator{}),
+					sdk.MsgTypeURL(&vestingtypes.MsgCreateVestingAccount{}),
+					sdk.MsgTypeURL(&ibctransfertypes.MsgTransfer{}),
+				},
+			}
+			app.ICAHostKeeper.SetParams(ctx, hostParams)
+
+			minGasPriceGenesisState := &globalfeetypes.GenesisState{
+				Params: globalfeetypes.Params{
+					MinimumGasPrices: sdk.DecCoins{sdk.NewDecCoinFromDec("uband", sdk.NewDecWithPrec(25, 4))},
+				},
+			}
+			app.GetSubspace(globalfee.ModuleName).SetParamSet(ctx, &minGasPriceGenesisState.Params)
```

Comment:
should we move this into the module (like initModule from icahost)? so, we can use am.paramSpace.SetParamSet there.